admin_manual/configuration/files/external_storage_configuration_gui.rst

```diff
@@ -166,3 +164,11 @@ You might need to setup a cron job that runs ``sudo -u www-data php occ files:sc
 (or replace "--all" with the user name, see also :doc:`../server/occ_command`)
 to trigger a rescan of the user's files periodically (for example every 15 minutes), which includes
 the mounted external storage.
+
+FTP
+---
+
+If you want to mount a FTP Storage, please install the app ``FTP Storage Support`` from ownCloud market.
```

Comment:
Can you change this sentence to reflect the first comment in the FTP/SFTP section above?